ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolSpecBuilder.java

```diff
@@ -316,6 +323,11 @@ public ProtocolSpec build(final HeaderBasedProtocolSchedule protocolSchedule) {
               privacyParameters.getPrivateStateGenesisAllocator());
     }
 
+    WithdrawalsValidator withdrawalsValidator = new WithdrawalsValidator.ProhibitedWithdrawals();
```

Comment:
NIcer to default to prohibited withdrawals instead of using a null check and doing it here